lib/msf/core/post/common.rb

```diff
@@ -10,7 +10,6 @@ def initialize(info = {})
           'Meterpreter' => {
             'Commands' => %w[
               stdapi_sys_config_getenv
-              stdapi_sys_process_close
```

Comment:
Keeping track of `process.close` statically was hard, so the tooling assumes that a process execution generally means there is close support in the meterpreter